lib/api.php

```diff
@@ -95,6 +95,13 @@ public function guid();
      * no-ops when in disabled mode.
      */
     public function disable();
+
+    /**
+     * Sets a list of options onto this trace
+     *
+     * @param array $options
+     */
+    public function options($options);
```

Comment:
My one hesitation is on this one: I'm not sure we want `options()` to be part of the public API. The reason being that there are some options that can *only* be set at initialization time -- and can't be changed at an arbitrary time on an Tracer instance. I want to be sure what whatever we support is made clear and we can *continue* to support those options over time. 

Changing the comment to `Currently for internal use only` might alleviate my concerns about what we're signing up to support :)